samples/snippets/src/main/java/com/example/speech/SpeechModelAdaptationBeta.java

```diff
@@ -0,0 +1,139 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_transcribe_with_model_adaptation_gcs]
+import com.google.cloud.speech.v1p1beta1.AdaptationClient;
+import com.google.cloud.speech.v1p1beta1.CreateCustomClassRequest;
+import com.google.cloud.speech.v1p1beta1.CreatePhraseSetRequest;
+import com.google.cloud.speech.v1p1beta1.CustomClass;
+import com.google.cloud.speech.v1p1beta1.CustomClass.ClassItem;
+import com.google.cloud.speech.v1p1beta1.LocationName;
+import com.google.cloud.speech.v1p1beta1.PhraseSet;
+import com.google.cloud.speech.v1p1beta1.PhraseSet.Phrase;
+import com.google.cloud.speech.v1p1beta1.RecognitionAudio;
+import com.google.cloud.speech.v1p1beta1.RecognitionConfig;
+import com.google.cloud.speech.v1p1beta1.RecognitionConfig.AudioEncoding;
+import com.google.cloud.speech.v1p1beta1.RecognizeResponse;
+import com.google.cloud.speech.v1p1beta1.SpeechAdaptation;
+import com.google.cloud.speech.v1p1beta1.SpeechClient;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionAlternative;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionResult;
+import java.util.List;
+
+public class SpeechModelAdaptationBeta {
+
+  public void transcribeWithModelAdaptation() throws Exception {
+    String uriPath = "gs://cloud-samples-tests/speech/brooklyn.flac";
+    String projectId =
+        "YOUR_PROJECT_ID"; // {api_version}/projects/{project}/locations/{location}/customClasses
+    String location = "YOUR_LOCATION"; // Region e.g. us-west1
+    transcribeWithModelAdaptation(projectId, location, uriPath);
+  }
+
+  /**
+   * Transcribe with model adaptation
+   *
+   * @param projectId your project id
+   * @param location the region
+   * @param gcsUri the path to the audio file
+   */
+  public static void transcribeWithModelAdaptation(String projectId, String location, String gcsUri)
+      throws Exception {
+    // Initialize the adaptation client that will be used to send requests.
+    try (AdaptationClient adaptationClient = AdaptationClient.create()) {
+
+      /*
+          Create`PhraseSet` and `CustomClasses` to create custom lists of similar
+          items that are likely to occur in your input data.
+      */
+
+      // The parent resource where the custom class and phrase set will be created.
+      LocationName parent = LocationName.of(projectId, location);
+
+      // Create the custom class
+      CreateCustomClassRequest classRequest =
+          CreateCustomClassRequest.newBuilder()
+              .setParent(parent.toString())
+              .setCustomClassId("customClassId1871032322")
```

Comment:
Does this specific value have some significant, or is it up to the user? Can either provide a comment with links to the format used, or change to something like `"YOUR-CUSTOM-CLASS-ID"` instead?